sdk/core/perf/inc/azure/perf/argagg.hpp

```diff
@@ -1436,7 +1436,7 @@ inline fmt_ostream::~fmt_ostream() { output << fmt_string(this->str()); }
 
 inline std::string lstrip(const std::string& text)
 {
-  auto result = text;
+  std::string result = text;
```

Comment:
I would have expected this to be the same.

Reply:
This silences a visual studio diagnostic complaining about the use of auto potentially introducing unwanted copies - I cleaned up all the other diagnostics and figured I'd silence this one as well.